Odata-docs/support/support-policy.md

```diff
@@ -0,0 +1,127 @@
+
+### OData.NET 5.x
+
+OData.NET 5.x refers to 5.x versions of the following libraries:
+
+- Microsoft.Data.OData
+- Microsoft.Data.Edm
+- System.Spatial
+- Microsoft.Data.Services.Client
+- Microsoft.Data.Services
+
+The libraries are currently in maintenance mode. There are no current plans to end support. There will be an official announcements at least 12 months before official support ends.
+
+The libraries support .NET Framework 4.0 and later.
+
+The libraries support OData proctocol versions 1 to 3.
+
+## AspNetCore OData
+
+This refers to the `Microsoft.AspNetCore.OData`  library which adds OData integration to ASP.NET Core applications.
+
+| Version     | OData.NET version | Original release date | Latest patch version  | Latest patch release date | Support phase | End of support |
+| ------------|-------------------|-----------------------|-----------------------|---------------------------|---------------|----------------|
+| 9.x | 8.x | TBD | 9.0.0-preview-2| 2024-06-05 | Preview | TBD |
+| 8.x | 7.x | 2021-07-03 | 8.2.5 |2023-03-01 | Active | TBD |
+| 7.x | 7.x | 2018-06-29 | 7.7.4 | 2024-01-29 | Active | TBD |
+
+Microsoft.AspNetCore.OData 9.x will be the next major release. This version will only support .NET 8 and later and OData.NET 8 and later.
+
+Microsoft.AspNetCore 8.x is currently actively supported. It supports .NET 5 and later and OData.NET 7.x.
+
+Microsoft.AspNetCore 7.x is currently actively supports. It supports .NET Framework 4.5 and later and .NET 5 and later. It supports OData.NET 7.x. It will enter maintenance mode soon.
+
+## AspNet OData
+
+This refers to the `Microsoft.AspNet.OData` library which adds OData integration to ASP.NET 4.x on .NET Framework.
+
+| Version     | OData.NET version | Original release date | Latest patch version  | Latest patch release date | Support phase | End of support |
+| ------------|-------------------|-----------------------|-----------------------|---------------------------|---------------|----------------|
+| 7.x | 7.x | 2018-06-29 | 7.7.4 | 2024-04-01 | Maintenance Mode | TBD |
+
+Microsoft.AspNet.OData 7.x is in maintenance phase. It will only receive security patches and critical bug fixes. There are no current plans to end support. Should there be plans to drop support, there will be an official announcement at least 12 months before official support ends. This version supports .NET Framework 4.5 and later.
+
+## OData ModelBuilder
+
+The `Microsoft.OData.ModelBuilder` library provides APIs that make it easier to create an IEdmModel from CLR (common language runtime) classes.
+
+| Version     | OData.NET version | Original release date | Latest patch version  | Latest patch release date | Support phase | End of support |
+| ------------|-------------------|-----------------------|-----------------------|---------------------------|---------------|----------------|
+| 1.x | 7.x and 8.x | 2020-09-02 | 1.0.9 | 2022-06-10 | Maintenance | 2025-06-27 |
```

Comment:
Will there be another release for this library once we end support for it? Should it be indicated here?

Reply:
I think this should still be in active support.